ui-design/virtual-machines/vm-details/vm-list.md

```diff
@@ -0,0 +1,42 @@
+# VM Details
+
+Clicking on the name of the Virtual Machine in the list view will take the user into the details view of that VM.
+
+![List of VMs](img/UI-VM-list.jpg)
+
+
+The details of the Virtual Machine are separated into tabs:
+- Overview
+- YAML
+- Snapshots
+- Networks
+- Disks
+- Consoles
+
+On the overview tab both a general overview and utilization are provided.
+
+![Virtual Machine Details](img/UI-VM-details.jpg)
+
+Additional information is provided via tooltips for status and boot order.
+![Virtual Machine Details](img/UI-VM-details-status-tooltip.jpg)
+
+The user can then edit the overall details of the VM from the action menu in the top right.
+
+![Virtual Machine Details](img/UI-VM-details-action-menu.jpg)
+
+Once in Edit mode all available options will show as editable.
+
+![Virtual Machine Details](img/UI-VM-details-edit-view.jpg)
+
```

Comment:
I'm not sure all options being editable makes sense. In particular:

* Pod shouldn't be editable, I don't know what the user would reasonably change it to.
* Project/Namespace shouldn't be editable, there is no action we can reasonably map this to on the backend to actually move objects from namespace to namespace.
* Flavour I'm assuming maps to presets? Something else?
* Not sure it makes sense to be able to toggle cloud-init on off all the time, as it usually would only apply the very first time the guest is booted.

Reply:
>   Pod shouldn't be editable, I don't know what the user would reasonably change it to.

+1, it should be a hyperlink to the pod

>    Project/Namespace shouldn't be editable, there is no action we can reasonably map this to on the backend to actually move objects from namespace to namespace.

+1, also should be hyperlink

>    Flavour I'm assuming maps to presets? Something else?

nope, this one: https://github.com/kubevirt/common-templates/pull/15 - it is a mechanism to utilize the openshift templates to bring common configuration (os/flavor/profile) as well as VM templates (the oVirt like templates).

>    Not sure it makes sense to be able to toggle cloud-init on off all the time, as it usually would only apply the very first time the guest is booted.

not sure. Is re-init not a valid use case?